src/components/navigator-bar/navigator-bar.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+  <v-bottom-navigation grow dark class="blue-grey darken-1 white--text">
+    <div class="flex-container-row flex-item-variable" style="justify-content: space-between">
```

Comment:
You can have a CSS class in `app.vue.ts` called `justify-content-space` and move this style there. If you want to use it, just add that class in the `class` attribute